src/utilities/formatDate.ts

```diff
@@ -15,9 +15,12 @@ const monthNames: { [index: string]: string } = {
   '12': i18n.t(langTokens.date.decemberGenitiveCase),
 };
 
-export const formatDate = (dateString = '', time = false): string => {
-  if (dateString.length) {
-    const fullDate = new Date(dateString);
+export const formatDate = (
+  dateValue: number | string = '',
+  time = false,
+): string => {
+  if (dateValue) {
```

Comment:
oh, I checked only now that there is `date-fns` library in the project(https://date-fns.org/v2.29.3/docs/format)

it will not help with replacing `monthNames` in Ukrainian, but besides that we can consume it to format rest of the string:

```
format(fullDate, `kk:mm dd ${monthNames[fullDate.getMonth() + 1]} yyyy`)
```

Reply:
thanks for the advice
updated in accordance